Sources/PostgresNIO/New/PSQLTask.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 25.01.21.
+//
+
+enum PSQLTask {
```

Comment:
Ditch Xcode template headers.
```suggestion
enum PSQLTask {
```